DataFormats/Detectors/TRD/include/DataFormatsTRD/TrackTRD.h

```diff
@@ -0,0 +1,35 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file TrackTRD.h
+/// \author David Rohr
+
+#ifndef O2_DATAFORMATS_TRACK_TRD_H
+#define O2_DATAFORMATS_TRACK_TRD_H
+
+#include "GPUTRDTrack.h"
+
+namespace o2
+{
+namespace trd
+{
+using TrackTRD = o2::gpu::GPUTRDTrack;
+} // namespace trd
+namespace framework
+{
+template <typename T>
+struct is_messageable;
+template <>
+struct is_messageable<o2::trd::TrackTRD> : std::true_type {
```

Comment:
I actually just saw the booleans here only now that Ruben mentioned them. There must not be booleans in data types that are transfered between host and GPU, since `sizeof(bool)` is implementation-defined. It is OK in local variables in functions, but not in data types that are exchanged. So that must be changed in any case.

Reply:
@martenole ok, not sure about the TRD PID, but for the propagation one should use the seeding mass hypothesis (or TRD should override it by setting the TrackPar::PID if its pid, e.g. for electron, is better).
In principle, this is handled automatically by the Propagator -> TrackParCov::correctForMaterials, which uses TrackPar::PID.
But the GPU track version has not PID info, afaik. Should it be added?

Reply:
For the GPU track version I let @davidrohr comment. I am not sure if it uses a mass hypothesis. For the CPU version the TRD track inherits from `TrackParametrizationWithError`, so the PID information of the seeding track is by construction available and therefore also used by the propagator.

Reply:
@martenole OK, I was referring to your  `but this is already fixed to the pion mass` which I guess is applies to GPU propagation. Indeed, better @davidrohr comments: it is true that for internal TPC propoagations the PID is irrelevant, but when going outside of TPC the energy loss does matter, so one should foresee mass assignment (and export to CPU version of the track).

Reply:
My comment that it is fixed to the pion mass referred also to the CPU propagation with AliRoot. In understand that the propagation was not ideal in that case. So for the GPU version I assume we should find a PID hypothesis before the extrapolation from TPC to TRD

Reply:
@sgorbuno can comment better, but as far as I see the GPU track model always assumes a pion: https://github.com/AliceO2Group/AliceO2/blob/dev/GPU/GPUTracking/Merger/GPUTPCGMPropagator.cxx#L890

Reply:
In the async phase, we run the TPC dEdx during the TPC track fit, so we could use that for the mass hypothesis without much additional effort.